Sources/docc/DocCDocumentation.docc/customizing-the-appearance-of-your-documentation-pages.md

```diff
@@ -79,7 +79,26 @@ of a green hue.
 
 As a general rule, the default color properties provided by DocC assumes a
 naming convention where "fill" colors are used for backgrounds and "figure"
-colors are used for foreground colors, like text.
+colors are used for foreground colors, like text. Note that colors defined in 
+`theme-settings.json` will be used across all pages of your documentation bundle.
+
+**Changing a color on for a specific article page**
+
+You can also use a ``PageColor`` metadata directive to set the fill color for an
+intro section of a specific article:
```

Comment:
The PageColor directive doesn't specifically set the fill color. It is an accent color that isn't associated to any specific on-page element. See [the PageColor documentation overview][1]. 

[1]: https://www.swift.org/documentation/docc/pagecolor#overview